terraform/resources/aca/locals.tf

```diff
@@ -0,0 +1,188 @@
+      cpu         = 0.5
+      memory      = "1Gi"
+      app_version = var.dibbs_version
+
+      is_public = false
+
+      env_vars = []
+
+      target_port = 8080
+    }
+    orchestration = {
+      name        = "orchestration"
+      cpu         = 0.5
+      memory      = "1Gi"
+      app_version = var.dibbs_version
+
+      is_public = true
+
+      env_vars = [
+        {
+          name  = "OTEL_METRICS",
+          value = "none"
+        },
+        {
+          name  = "OTEL_METRICS_EXPORTER",
+          value = "none"
+        },
+        {
+          name  = "INGESTION_URL",
+          value = "http://ingestion.internal.${azurerm_container_app_environment.ce_apps.default_domain}"
+        },
+        {
+          name  = "VALIDATION_URL",
+          value = "http://validation.internal.${azurerm_container_app_environment.ce_apps.default_domain}"
+        },
+        {
+          name  = "FHIR_CONVERTER_URL",
+          value = "http://fhir-converter.internal.${azurerm_container_app_environment.ce_apps.default_domain}"
+        },
+        {
+          name  = "ECR_VIEWER_URL",
+          value = "http://ecr-viewer.${azurerm_container_app_environment.ce_apps.default_domain}/ecr-viewer"
+        },
+        {
+          name  = "MESSAGE_PARSER_URL",
+          value = "http://message-parser.internal.${azurerm_container_app_environment.ce_apps.default_domain}"
+        },
+        {
+          name  = "TRIGGER_CODE_REFERENCE_URL",
+          value = "http://trigger-code-reference.internal.${azurerm_container_app_environment.ce_apps.default_domain}"
+        }
+      ]
+
+      target_port = 8080
+    }
+    validation = {
+      name        = "validation"
+      cpu         = 0.5
+      memory      = "1Gi"
+      app_version = var.dibbs_version
+
+      is_public = false
+
+      env_vars = []
+
+      target_port = 8080
+    }
+    trigger-code-reference = {
+      name        = "trigger-code-reference"
+      cpu         = 0.5
+      memory      = "1Gi"
+      app_version = var.dibbs_version
+
+      is_public = false
+
+      env_vars = []
+
+      target_port = 8080
+    }
+    ecr-viewer = {
+      name        = "ecr-viewer"
+      cpu         = 0.5
+      memory      = "1Gi"
+      app_version = var.dibbs_version
+
+      is_public = true
+
+      env_vars = [
+        {
+          name  = "AZURE_STORAGE_CONNECTION_STRING",
+          value = var.azure_storage_connection_string
+        },
+        {
+          name  = "AZURE_CONTAINER_NAME",
+          value = var.azure_container_name
+        },
+        {
+          name  = "NEXT_PUBLIC_NON_INTEGRATED_VIEWER",
+          value = false
+        },
+        {
+          name  = "SOURCE",
+          value = "azure"
+        },
+        {
+          name  = "APP_ENV",
+          value = "test"
+        },
+        {
+          name  = "NODE_ENV",
+          value = "production"
+        },
+        {
+          name  = "NEXT_PUBLIC_BASEPATH",
+          value = "/ecr_viewer"
```

Comment:
I'd love to see some of these as variables that can be overwritten, but we don't need to push on that for now.